<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_checkInfo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>191</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>检测结果：</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>30</y>
       <width>49</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_rtc">
      <item>
       <widget class="QLabel" name="label_lightpic_rtc">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_rtc">
        <property name="text">
         <string>rtc</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>50</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_uart232">
      <item>
       <widget class="QLabel" name="label_lightpic_uart232">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_uart232">
        <property name="text">
         <string>uart232</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>70</y>
       <width>85</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_flash">
      <item>
       <widget class="QLabel" name="label_lightpic_flash">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_flash">
        <property name="text">
         <string>spiFlash</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>90</y>
       <width>80</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_lora">
      <item>
       <widget class="QLabel" name="label_lightpic_lora">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_lora">
        <property name="text">
         <string>spiLora</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>110</y>
       <width>75</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_at88">
      <item>
       <widget class="QLabel" name="label_lightpic_at88">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_at88">
        <property name="text">
         <string>at88sc</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>130</y>
       <width>65</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_stm8">
      <item>
       <widget class="QLabel" name="label_lightpic_stm8">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_stm8">
        <property name="text">
         <string>stm8</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>150</y>
       <width>49</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4g">
      <item>
       <widget class="QLabel" name="label_lightpic_4g">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_4g">
        <property name="text">
         <string>4g</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>170</y>
       <width>63</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_lan1">
      <item>
       <widget class="QLabel" name="label_lightpic_lan1">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_lan0">
        <property name="text">
         <string>Lan1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>190</y>
       <width>63</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_lan2">
      <item>
       <widget class="QLabel" name="label_lightpic_lan2">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_lan1">
        <property name="text">
         <string>Lan2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>210</y>
       <width>63</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_lan3">
      <item>
       <widget class="QLabel" name="label_lightpic_lan3">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_lan2">
        <property name="text">
         <string>Lan3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>230</y>
       <width>63</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_lan4">
      <item>
       <widget class="QLabel" name="label_lightpic_lan4">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_lan3">
        <property name="text">
         <string>Lan4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>250</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_wan0">
      <item>
       <widget class="QLabel" name="label_lightpic_wan0">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_wan0">
        <property name="text">
         <string>Wan0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>270</y>
       <width>56</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_wifi">
      <item>
       <widget class="QLabel" name="label_lightpic_wifi">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_wifi">
        <property name="text">
         <string>wifi</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>290</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_ntp">
      <item>
       <widget class="QLabel" name="label_lightpic_ntpClient">
        <property name="text">
         <string>@ </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_light_ntpClient">
        <property name="text">
         <string>ntpdate </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_lightpic_agingResult">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>310</y>
       <width>18</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>@ </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_light_aging">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Uptime               </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>671</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_notes">
     <item>
      <widget class="QLabel" name="label_notes">
       <property name="text">
        <string>提示信息：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_notes">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_light_keyPressed">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>@ whole</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_checkStatus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>671</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_serial">
     <item>
      <widget class="QLabel" name="label_serialNum">
       <property name="text">
        <string>序列号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_serialNum">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>110</y>
      <width>191</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_version">
     <item>
      <widget class="QLabel" name="label_uptime">
       <property name="text">
        <string>currentUptime:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_VersionAndRunCnt">
       <property name="text">
        <string>checkApp Version</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>系统提示:</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>290</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_ipWifi">
        <property name="text">
         <string>Wifi( ip ):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_ipWifi">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>271</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSysTime">
        <property name="text">
         <string>系统时间：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEditSys"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelCpuTemp">
        <property name="text">
         <string>cpu温度：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_bar_cpuTemp">
        <property name="text">
         <string>tempBar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_celsius">
        <property name="text">
         <string>Celsius</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1239</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
